stylelint-config/rules/use-defined-css-var-when-available/props-to-allowed-vars.js

```diff
@@ -14,46 +14,6 @@ const BORDER_WIDTHS = ["--border-width"];
 
 const BORDER_STYLES = ["--border-style"];
 
-const FONT_SIZES = [
```

Comment:
I'm not exactly sure what this file is for and there's a lot of mixture between relevant stuff and irrelevant stuff, so please help me understand it better